documentation/01-Prerequisites.md

```diff
@@ -138,9 +133,7 @@ sudo apt-get install liblttng-ctl0 liblttng-ust0 zlib1g libxml2
 ```
 
 > Note: older distros such as Ubuntu 16.04 may also need the `libcurl3` package installed for `dotnet restore` to work, but is unavailable on (or will break) more recent versions of Ubuntu.
-
 > Note: `libxml2` pulls an appropriate version of `libicu`.
-
```

Comment:
We should either leave the extra new line separators or else add a trailing \ to each preceeding line, else each of these comments wraps into the other.

Reply:
Same comment in several other places.

Reply:
Another alternative is to make the blank line have a leading `>` .  That renders them all as the same block, but I think it's ok.

https://github.com/DavidAnson/markdownlint/blob/v0.20.4/doc/Rules.md#md028

Reply:
I'm not seeing a way to offer a "suggestion" for automatically improving the PR via the webpage for this issue.

Reply:
If you want to skip this one for now and just merge that's fine with me.  I can fix it up later.

Reply:
fixed it, I believe

Reply:
LGTM: https://github.com/microsoft/MLOS/blob/2cb73276287797b059adc6c3d1caeceff321119d/documentation/01-Prerequisites.md#option-2-manual-build-tools-install
Thanks!